### PR TITLE
Travis updates

### PR DESCRIPTION
**Pull request by markscamilleri**
Updated travis to expand a build matrix for both frontends. (PR will test it out)

### CODE REVIEW

**Comment by HarryPi** [comment]
Right please let me know when u want me to review this then

**Comment by markscamilleri** [comment]
Build is failing because of AdminClient.. This is boilerplate code which is being removed in #34

**Comment by markscamilleri** [comment]
/AdminClient can be ignored as this is all changing. It's currently boilerplate code just to make it pass checks

File: AdminClient/package.json [hunk 1]
```diff
@@ -11,37 +11,42 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~8.2.14",
```
Comment (DominikWas) [inline AdminClient/package.json]: good version updates, but does this mean we need a minimum version of these dependencies?

File: AdminClient/src/app/auth/auth.page.ts [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  constructor(
+    public menuController: MenuController,
+    public loadingController: LoadingController,
+    private store: Store<AppState>,
+    private snackBar: MatSnackBar,
+  ) {
+
+  }
+
+  ionViewWillEnter(): void {
```
Comment (DominikWas) [inline AdminClient/src/app/auth/auth.page.ts]: just curious, what's the reasoning behind using ionic for web here?

File: AdminClient/package.json (hunk 1 above)
Comment (markscamilleri) [reply AdminClient/package.json]: Yes, cause of angular-cli being updated to version 9, everything angular in the admin client will be updated to version 9.

File: AdminClient/src/app/auth/auth.page.ts (hunk 2 above)
Comment (markscamilleri) [reply AdminClient/src/app/auth/auth.page.ts]: Copy and Pasted to have the tests pass.

This is being removed in #34

**Comment by markscamilleri** [comment]
Cannot merge this for now as I need to get the workflow for the backend from @travelingdeveloper12

**Comment by markscamilleri** [comment]
I removed the backend tests. These must be added before the first backend merge

**Pull request merged**